EMML/1001-2000/EMML1832.xml

```diff
@@ -45,35 +146,930 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
+                        <locus target="#35v"/>
+                        <desc>Miniature depicting <persName ref="PRS6902Matthew"/>.</desc>
+                        <q xml:lang="gez">ቅዱስ፡ ማቴዎስ። ።</q>
+                     </decoNote>
+                     <decoNote xml:id="d27" type="miniature">
+                        <locus target="#132v"/>
+                        <desc>Miniature depicting <persName ref="PRS6751Mark"/>.</desc>
+                        <q xml:lang="gez">ቅዱስ፡ ማርቆስ። ።</q>
+                     </decoNote>
+                     <decoNote xml:id="d28" type="miniature">
+                        <locus target="#184v"/>
+                        <desc>Miniature depicting <persName ref="PRS6387Luke"/>.</desc>
+                        <q xml:lang="gez">ቅዱስ፡ ሉቃስ። ።</q>
+                     </decoNote>
+                     <decoNote xml:id="d29" type="miniature">
+                        <locus target="269v"/>
+                        <desc>Miniature depicting <persName ref="PRS5695John"/>.</desc>
+                        <q xml:lang="gez">ቅዱስ፡ ዮሐንስ። ።</q>
+                     </decoNote>
+                     <decoNote xml:id="d30" type="unspecified">
+                        <desc>Beginning and end of text often marked with crux ansata or other decorative signs throughout.</desc>                        
                      </decoNote>
                   </decoDesc>
+                  <additions>
+                     <list>
+                        <item xml:id="a1">
+                           <locus target="#1r"/>
+                           <desc type="LandGrant">Poorly legible record of a land grand by <persName ref="PRS3874eskender"/> to 
+                              <persName ref="PRS14233TawaldaM"/> in <date notBefore="1488" notAfter="1499">6981 (= 1488/9 A.D.)</date>, 
+                              when the ʿaqqābe saʿat was <persName ref="PRS14232TaklaIyasusMoa"/>. 
+                              The last lines of the note are not visible behind a sheet of paper stuck to the page.</desc>
+                           <q xml:lang="gez">በአኰቴተ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ወሀቦ፡ አነ፡ <persName ref="PRS3874eskender">እስክንድር፡</persName>
+                              ወስመ፡ <supplied reason="lost">መን</supplied>ግሥትየ፡ 
+                              <persName ref="PRS3874eskender"><supplied reason="lost">ቍ</supplied>ስጠንጢኖስ፡</persName> 
+                              <gap reason="ellipsis" unit="lines" quantity="9"/></q>
+                        </item>
+                        <item xml:id="a2">
+                           <locus from="1va" to="1vb"/>
+                           <desc type="LandGrant">Record of a land grant by ʿaqqābe saʿat <persName ref="PRS14065Tewogolos"/>
+                              to marāḥi <persName ref="PRS9199TaklaYo">Takla Yoḥannǝs</persName> in compensation for the 
+                              land that had been troyed during the reign of <persName ref="PRS6229LebnaDe"/>.</desc>
```

Comment:
destroyed?

Reply:
The term is from Getatchews description. I think, it means "plundered". I may change it to avoid confusion.

Reply:
There is no such word in English, must be a typo wherever you saw it.

Reply:
Ok, so I will change to "destroyed".